ambry-router/src/test/java/com/github/ambry/router/QuotaAwareOperationControllerTest.java

```diff
@@ -107,6 +108,9 @@ public void setupMocks() throws Exception {
     quotaAwareOperationController =
         new QuotaAwareOperationController(null, null, null, networkClientFactory, clusterMap, routerConfig, null, null,
             routerMetrics, null, null, null, null, nonBlockingRouter);
+    Field privateField = OperationController.class.getDeclaredField("putManager");
+    privateField.setAccessible(true);
+    ((PutManager) privateField.get(quotaAwareOperationController)).close();
```

Comment:
can we just expose PutManage as a package-level accessible object in QuotaAwareOperationController. This is convoluted.

Reply:
It was already package accessible. cleaned it up